src/Hl7.Fhir.Specification.Tests/StructureDefinitionSerializationInfoTests.cs

```diff
@@ -11,7 +11,7 @@ public class StructureDefinitionSerializationInfoTests
         [ClassInitialize]
         public static void SetupSource(TestContext t)
         {
-            source = ZipSource.CreateValidationSource();
+            source = new MultiResolver(ZipSource.CreateValidationSource(), new DirectorySource("TestData", includeSubdirectories: true));
```

Comment:
To construct DirectorySource like this is obsolete: use 
`new DirectorySource("TestData", new DirectorySourceSettings(includeSubdirectories: true))`

Reply:
Changed as requested